Dockerfile

```diff
@@ -0,0 +1,29 @@
+#Dockerfile is used to create an image
+
+#Set the base image.
+FROM python:3.9-slim
+# This is for interacting with a PostgreSQL database; we aren't doing that but one of my dependencies listed in
+# requirements.txt requires libpq-dev, so it must be included.
+RUN apt-get update
+RUN apt-get install -y gcc libpq-dev
+
+#Set the work directory in the container
+WORKDIR /home
+
+# copy the dependencies file to the working directory
+COPY requirements.txt .
+
+# Ensure pip is up to date
+RUN pip install --upgrade pip
+
+# install dependencies
+RUN pip install -r requirements.txt
```

Comment:
I am obligated to ask if you'd consider using poetry instead of a requirements.txt 😅 Is this meant to be a repo to act as a template for other repos? Or is this a pure example of how it works on Libera?